cantabular/dimensions.go

```diff
@@ -148,3 +148,29 @@ func (c *Client) GetDimensionOptions(ctx context.Context, req GetDimensionOption
 
 	return &resp.Data, nil
 }
+
+func (c *Client) GetAreas(ctx context.Context, req QueryData) (*GetAreasResponse, error) {
+	resp := &struct {
+		Data   GetAreasResponse `json:"data"`
+		Errors []gql.Error      `json:"errors,omitempty"`
+	}{}
+
+	data := QueryData{
+		Dataset: req.Dataset,
+		Text:    req.Text,
+	}
+
+	if err := c.queryUnmarshal(ctx, QueryAreasByArea, data, resp); err != nil {
+		return nil, err
```

Comment:
speaking with Josh on my ticket, he was keen to always wrap the error with a contextual message. Not sure if you want to do that.
Non-blocker.

Reply:
Yes, please when returning errors can we always make sure to wrap with a brief description of what has failed.. Historically we've done this with either `fmt.Errorf` or `errors.Wrap` (`github.com/pkg/errors`), now moving to only using `errors.Wrap` in order to retain a stack trace. Example:
```
return nil, errors.Wrap(err, "failed to unmarshal query")
```

Reply:
I've updated this and also updated the tests to reflect the new way of wrapping errors